health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -331,11 +331,11 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
           <PopUp
             className={"popUpClass"}
             type={"default"}
-            heading={t("CONFIRM_TO_DELETE")}
+            heading={t("FOR_CONFIRM_TO_DELETE")}
             equalWidthButtons={true}
             children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)